ocs_samples/basic_samples/SDS/DotNet/SdsClientLibraries/SdsClientLibraries/Program.cs

```diff
@@ -223,15 +218,26 @@ public static async Task<bool> MainAsync(bool test = false)
                 // We will retrieve events filtered to only get the ones where the radians are less than 50.  Note, this can be done on index properties too.
 
                 var retrievedInterpolatedFiltered = (await dataService.GetWindowFilteredValuesAsync<WaveData>(stream.Id, "0", "180", SdsBoundaryType.ExactOrCalculated, "Radians lt 50"));
-                Console.WriteLine(" Sds will only return the values where the radains are less than 50:");
+                Console.WriteLine(" Sds will only return the values where the radians are less than 50:");
```

Comment:
Being nit-picky here, but now that you made it more readable I noticed this extra space here... :)
Also below you added "SDS can return a sample of your data population to show trends." where SDS is all in caps and that's the way it's supposed to be but do you mind fixing all these other ones too so they all match... like this Sds on this specific line is not all in caps...

Reply:
Standalone "SDS" console output should now be consistent across the three Program.* files.
And I removed that space.